lib/web_ui/lib/src/engine/text/font_collection.dart

```diff
@@ -266,18 +289,13 @@ class _PolyfillFontManager extends FontManager {
 
     paragraph.style.fontFamily = "'$family', $fallbackFontName";
 
-    final Completer<void> completer = Completer<void>();
```

Comment:
we still need this completer so we can actually wait for the fonts to load in the `_PolyfillFontManager`.